Tasks/google-play-release/task.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "8cf7cac0-620b-11e5-b4cf-8565e60f4d27",
-    "name": "google-play-release",
+    "name": "GooglePlayRelease",
```

Comment:
It would be great if it was renamed a lot more convenience when testing. If renaming doesn't break anything I would cool with the new name.